src/frontend/src/lib/components/exchange/ExchangeBalance.svelte

```diff
@@ -2,12 +2,10 @@
 	import { formatUSD } from '$lib/utils/format.utils';
 	import { exchangeInitialized } from '$lib/derived/exchange.derived';
 	import { combinedDerivedSortedNetworkTokensUi } from '$lib/derived/network-tokens.derived';
+	import { getTokensTotalUsdBalance } from '$lib/utils/tokens.utils';
 
 	let totalUsd: number;
-	$: totalUsd = $combinedDerivedSortedNetworkTokensUi.reduce(
-		(acc, token) => acc + (token.usdBalance ?? 0),
-		0
-	);
+	$: totalUsd = getTokensTotalUsdBalance($combinedDerivedSortedNetworkTokensUi);
```

Comment:
it's not really a getter per sé, maybe we can rename it to something like `sumTotalUsdBalance` or better idea?

Reply:
Agreed. 
I renamed it to `sumTokensUsdBalance` to highlight that a list of tokens is expected as param and also removed the `Total` part since `sum` already means that there will be a total. WDYT?

Reply:
Coolio, thanks.